ArduCopter/Copter.h

```diff
@@ -439,10 +408,11 @@ class Copter : public AP_HAL::HAL::Callbacks {
     // Attitude, Position and Waypoint navigation objects
     // To-Do: move inertial nav up or other navigation variables down here
 #if FRAME_CONFIG == HELI_FRAME
-    AC_AttitudeControl_Heli attitude_control;
+    #define AC_AttitudeControl_t AC_AttitudeControl_Heli
 #else
-    AC_AttitudeControl_Multi attitude_control;
+    #define AC_AttitudeControl_t AC_AttitudeControl_Multi
 #endif
+    AC_AttitudeControl_t attitude_control;
```

Comment:
@lucasdemarchi multi vs heli is all done at compile time ATM.  I don't know if that can/should be changed, but I'm pretty sure it is beyond the scope of this patch set :-)

Given that, and that I use this type definition (which could actually be a typedef....) elsewhere to create a reference to the heli-style object, I think it is required.